updating swiftlint content (#6073)
kamranahmedse/developer-roadmap: src/data/roadmaps/ios/content/swiftlint@NnTC3R8goZQTXHmfHKHbM.md

> # SwiftLint
> 
> SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.
> 
> Learn more from the following resources:
> 
> Now that you have successfully installed SwiftLint, learn the next steps and more from the following resources:
> 
> - [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
> - [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)

Reviewer: ```suggestion
# SwiftLint

SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.

Learn more from the following resources:

- [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
- [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)
```

> # SwiftLint
> 
> SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.
> 
> Learn more from the following resources:
> 
> Now that you have successfully installed SwiftLint, learn the next steps and more from the following resources:
> 
> - [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
> - [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)

Reviewer: ```suggestion
# SwiftLint

SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.

Learn more from the following resources:

Now that you have successfully installed SwiftLint, learn the next steps and more from the following resources:

- [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
- [@video@Mastering SwiftLint for Code Readability](https://www.youtube.com/watch?v=4YQ6DJ-xovY)
```

> # SwiftLint
> 
> SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.
> 
> Learn more from the following resources:
> 
> Now that you have successfully installed SwiftLint, learn the next steps and more from the following resources:
> 
> - [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
> - [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)

Reviewer: ```suggestion
# SwiftLint

SwiftLint is an open-source tool developed by Realm, that facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.

Learn more from the following resources:

- [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
- [@video@Mastering SwiftLint for Code Readability](https://www.youtube.com/watch?v=4YQ6DJ-xovY)
```

# SwiftLint

SwiftLint is an open-source tool developed by Realm, facilitates maintaining standardized code development by enabling the definition and enforcement of project-specific conventions and guidelines. It ensures developers adhere to these standards, thereby promoting organized code practices.

> You have many options to install SwiftLint, for this tutorial, we’ll use Homebrew.
> 
> ### Installing SwiftLint With Homebrew
> 
> If you don't have Homebrew installed yet, you can easily do so by running the following command in your Terminal:
> 
> ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
> 
> Once Homebrew is installed, proceed to install the latest version of SwiftLint with the following command:
> 
> ```brew install swiftlint```
> 
> ### Further Resources
> 
> Now that you have sucessfully installed SwiftLint, learn the next steps and more from the following resources:
> 
> - [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
> - [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)

Reviewer: ```suggestion
Learn more from the following resources:

Now that you have successfully installed SwiftLint, learn the next steps and more from the following resources:

- [@official@SwiftLint Documentation](https://github.com/realm/SwiftLint)
- [@video@Mastering SwiftLint for Code Readability)](https://www.youtube.com/watch?v=4YQ6DJ-xovY)
```

Reviewer: Since the first link contains installation instructions, there's no need to add it to the copy :)

I've also added "learn more from the following resources:" which is our standard line prior to links.
